### ISSUE TITLE
Geocoder does not work

### ISSUE THREAD

**Issue by zhishenh** [sebastianbaar/cordova-plugin-nativegeocoder#32]
I tried to add nativegeocoder in my Ionic App and executed the reverseGeocode function. In the browser, it showed that cordova is not available. Then I run my app in android device, it showed that Geocoder is not present on this device/emulator. How can I fix this problem?

**Comment by sebastianbaar** [comment]
Hey @zhishenh,
**TL;DR**: you can't. 

**But Why?**: The Plugin checks for `Geocoder.isPresent()` ([Android Docs](https://developer.android.com/reference/android/location/Geocoder.html#isPresent())). Geocoder simply won't work on this device. One reason for this could be that you are running your app on an emulator or on a device without Google Play Services but Geocoder API "[...] requires a backend service that is not included in the core android framework".

**Any workaround?**: Yes. Query [Google Maps Geocoding API](https://developers.google.com/maps/documentation/geocoding/start) as a backup if Geocoder is not present.

**Can we implement this backup for the plugin?**: No. I decided not to implement Google Maps Geocoding API as a backup because you have to add an [API key and enable billing](https://developers.google.com/maps/documentation/geocoding/usage-and-billing). I want this plugin to remain as small and lightweight as possible.

Cheers!
